frontend/src/container/GridGraphLayout/Graph/FullView/index.tsx

```diff
@@ -126,13 +129,11 @@ const FullView = ({
 		onFetchDataHandler();
 	}, [onFetchDataHandler]);
 
-	if (state.error && !state.loading) {
-		return (
-			<NotFoundContainer>
-				<Typography>{state.errorMessage}</Typography>
-			</NotFoundContainer>
-		);
-	}
+	//antd notification config
+	notification.config({
```

Comment:
can we use notification.success || notification.error only

Reply:
If we want to show any success message than we can use that also. So, currently as we want to show error only that's why I have used notification.error

Reply:
@NishidhJain we are using the default config. so i don't think there is a need for over riding the notification config

we can use `notification.success || notification.error`

Reply:
@pal-sig  Okay, so you are saying that I should remove this 'notification.config' part

Reply:
Yes @NishidhJain

Reply:
Please use the default config

Reply:
@pal-sig Now it is using the default configuration, I have removed the code for notification configuration